feedback.md

```diff
@@ -0,0 +1,24 @@
+# Video Store
+## What We're Looking For
+
+| Feature | Feedback    |
+| :------------- | :------------- |
+| **Core Requirements** |   |
+| Git hygiene |  |
+| Comprehension questions	|  |
+| Link to Trello |  |
```

Comment:
I think we want these in the comprehension questions right? So the students can fill them in